habitat_sim/utils/viz_utils.py

```diff
@@ -94,3 +353,83 @@ def display_video(video_file: str, height: int = 400):
         else:
             opener = "open" if sys.platform == "darwin" else "xdg-open"
             subprocess.call([opener, video_file])
+
+
+def place_agent(sim, pos=[0.0, 0.0, 0.0], rot=np.quaternion(-1, 0, 0, 0)):
+    """    
+    Places our agent in the scene using specified position and orientation parameters
+
+    Parameters
+    ----------
+    sim : Simulator, 
+        Reference to the habitat simulator object
+    pos : List, optional
+        Location in scene to place agent. The default is [0.0,0.0,0.0].
+    rot : numpy quaternion, optional
+        Orientation of agent in scene.  The default is np.quaternion(-1, 0, 0, 0)
+
+    Returns
+    -------
+    agent's scene_node transformation matrix
+
+    """
+    agent_state = habitat_sim.AgentState()
+    agent_state.position = pos
+    agent_state.rotation = rot
+    agent = sim.initialize_agent(0, agent_state)
+    return agent.scene_node.transformation_matrix()
+
+
+def remove_all_objects(sim):
```

Comment:
This is general enough we may want to add a helper to Simulator.cpp, thoughts @aclegg3 ?